Source/Utils/Average.H

```diff
@@ -133,11 +133,11 @@ namespace Average{
         return volfrac * c;
     };
 
-    void FineToCoarse ( MultiFab& mf_cp,
-                        const MultiFab& mf_fp,
-                        const int scomp,
-                        const int ncomp,
-                        const IntVect ratio );
+    void Coarsen ( MultiFab& mf_cp,
```

Comment:
would this (or above) be a good location for an extended doxygen string for the method?

Reply:
Yes, thanks. This PR is still very much in progress as we are in the process of deciding what we want to implement with @MaxThevenet and @RevathiJambunathan. Once the implementation is completed I will add a Doxygen documentation for the new method(s) in this header file.

Reply:
Sounds awesome, thx! :)